app/components/support_interface/tile_component.rb

```diff
@@ -1,12 +1,17 @@
 module SupportInterface
   class TileComponent < ViewComponent::Base
-    attr_reader :count, :label, :colour
+    attr_reader :count, :label, :colour, :href
 
-    def initialize(count:, label:, colour: :default, size: :regular)
+    def initialize(count:, label:, colour: :default, size: :regular, href: nil)
       @count = count
       @label = label
       @colour = colour
       @size = size
+      @href = href
```

Comment:
Consider changing this to `anchor` so it's more indicative as to how it's used.

Reply:
Hmmm, it’s used as an anchor on the performance dashboard, but can take any given URL, so maybe `href` is clearer? Or is `anchor` a Rails convention?

Reply:
No it's not. I simply did not see any cases where it's used to link to anything but anchors so it feels like premature optimisation. Up to you.

Reply:
I’ll leave it for now… I think it’s a bit clearer, but that’s only with my HTML hat on!